doc/api/repl.md

```diff
@@ -78,15 +78,16 @@ The default evaluator supports direct evaluation of JavaScript expressions:
 ```js
 > 1 + 1
 2
-> var m = 2
+> const m = 2
 undefined
 > m + 1
 3
 ```
 
 Unless otherwise scoped within blocks (e.g. `{ ... }`) or functions, variables
-declared either implicitly or using the `var` keyword are declared at the
-`global` scope.
+declared either implicitly or using the `const` or `let` keywords
+are declared at the `global` scope (as well as with the `var` keyword
+outside of functions).
 
```

Comment:
But it is exactly this remark —  "Unless otherwise scoped within blocks or functions" —  that prevents placing `const`, `var`, and `let` in the same list: `var` is not block scoped. Maybe you or somebody else could propose the whole clear concise sentence there? I am not a native speaker, so I am a bit clumsy there.